openstates/oh/bills.py

```diff
@@ -13,43 +13,333 @@
+            vote_results = {"approved":True,
+                            "passed":True,
+                            "adopted":True,
+                            "true":True,
+                            "false":False,
+                            True:True,
+                            False:False}
+
+            action_dict = {"ref_ctte_100":"committee:referred",
+                            "intro_100":"bill:introduced",
+                            "pass_300":"bill:passed",
+                            "intro_110":"bill:reading:1",
+                            "refer_210":"committee:referred",
+                            "crpt_301":"other",
+                            "crpt_317": "other",
+                            "intro_102":["bill:introduced","bill:passed"],
+                            "intro_105":["bill:introduced","bill:passed"],
+                            "intro_ref_ctte_100":"committee:referred",
+                            "refer_209":"other",
+                            "intro_108":["bill:introduced","bill:passed"],
+                            "intro_103": ["bill:introduced","bill:passed"],
+                            "msg_reso_503": "bill:passed",
+                            "intro_107": ["bill:introduced","bill:passed"],
+                            "imm_consid_360": "bill:passed"}
+
+            
+
+            base_url = "http://search-prod.lis.state.oh.us/"
+            first_page = base_url + "/solarapi/v1/general_assembly_{session}/".format(session=session)
+            legislators = self.get_legislator_ids(first_page)
+            all_amendments = self.get_other_data_source(first_page,base_url,"amendments")
+            all_fiscals = self.get_other_data_source(first_page,base_url,"fiscals")
+            all_synopsis = self.get_other_data_source(first_page,base_url,"synopsiss")
+            all_analysis = self.get_other_data_source(first_page,base_url,"analysiss")
+            doc_types = ["bills","resolutions"]
+            for doc_type in doc_types:
+                bill_versions = {}
+                for doc in self.pages(base_url,first_page+doc_type):
+                    for v in doc["items"]:
+                        #bills is actually a list of versions
+                        #going to create a dictionary as follows:
+                        #key=bill_id
+                        #value = dict of all versions, where keys are versionids
+                        #and values are the bill data from the api.
+                        #then loop through that to avoid duplicate bills
+
+                        bill_id = v["number"]
+                        version_id = v["versionid"]
+                        if bill_id in bill_versions:
+                            if version_id in bill_versions[bill_id]:
+                                self.logger.warning("There are two versions of {bill_id} called the same thing. Bad news bears!".format(bill_id=bill_id))
```

Comment:
I think it'd be worth raising an error instead of just a warning here just since version name duplication would make the bill's data really unclear (and is also very unlikely to happen). But a not-necessary change, for sure.